extensions/investspec/internationalization/messages.pot

```diff
@@ -0,0 +1,124 @@
+# Translations template for InVEST.
+# Copyright (C) 2021 Natural Capital Project
+# This file is distributed under the same license as the InVEST project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
+#
```

Comment:
Do we want both this file, `messages.pot` and the above file `messages.po`? I think I read in one of your Readmes that we can only need to check one in because the other is easily derived? Maybe it's different for this case? Sorry if I'm not fully understanding or confusing something!

Reply:
I think you're thinking of the `messages.mo` that we don't need to include... those are binary "compiled" versions of the PO files. The POT and PO files should both be checked in because the POT file is the source of truth for the current set of translated strings, while the PO files store the translations for each language.